### PR TITLE
Add missing hover and selected state colors to Widgets

### PR DESCRIPTION
**Pull request by jesusbotella**
Related to #640 

- Deep-insights.js https://github.com/CartoDB/deep-insights.js/pull/641
- CartoDB https://github.com/CartoDB/cartodb/pull/13275

----

This PR adds the missing hover and selected state colors to Widgets.
Based on https://user-images.githubusercontent.com/14546701/32537554-50cc906e-c463-11e7-9e20-5c11a5ce4a32.png

As I removed `tinycolor` dependency and it is not used anywhere else, do you want me to remove it from the project?

#### Category Widgets

- Check that `Bar Color Green with 16% black` color is applied when mouse is over any item
- Check that `Bar Color Dark` is applied when any item is selected

#### Histogram Widgets

- Check that `Bar Color Green with 16% black` color is applied when mouse is over any bar
- Check that `Bar Color Dark` is applied when there is a range selected

### CODE REVIEW

**Comment by jesusbotella** [comment]
@urbanphes I noticed that the `Other` category in the example has a solid background when disabled, and right now in the Builder is looks like this:

screen shot 2017-12-18 at 18 20 03

Do you want me to change it?

File: src/widgets/histogram/chart.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ var _ = require('underscore');
 var d3 = require('d3');
 var d3Interpolate = require('d3-interpolate');
 var cdb = require('cartodb.js');
-var tinycolor = require('tinycolor2');
```
Comment (rubenmoya) [inline src/widgets/histogram/chart.js]: Is it used anywhere else in the project? If not we could remove it from the `package.json`

Comment (jesusbotella) [reply src/widgets/histogram/chart.js]: I think it is not used anywhere else, I searched and there were only two occurrences: https://github.com/CartoDB/deep-insights.js/search?utf8=%E2%9C%93&q=tinycolor&type=

So I am going to remove it from package.json

**Comment by urbanphes** [comment]
@jesusbotella sorry. We let "OTHER" bar with the patterns styles a production.

captura de pantalla 2017-12-21 a las 9 41 16

**Comment by matallo** [comment]
then back to development!

**Pull request merged**